plugins/modules/efs.py

```diff
@@ -79,6 +79,12 @@
         description:
             - If the throughput_mode is provisioned, select the amount of throughput to provisioned in Mibps.
         type: float
+    backup:
```

Comment:
Can you call it `backup_policy` to be consistent with AWS's naming.

Reply:
This makes sense, I will change this